lua/entities/gmod_tardis/modules/teleport/sh_tp_failed.lua

```diff
@@ -0,0 +1,215 @@
+-- Failed teleport, interrupted teleport, effects
```

Comment:
Comment may not be accurate now as interrupted teleport is now in a different file?

Reply:
Also, we should also have little comments at the top of each file in here, some appear to be missing them